client/lib/feed-store/index.js

```diff
@@ -89,6 +94,17 @@ function receiveFeeds( newFeeds ) {
 	} ) );
 }
 
+function discoverFeeds( data, feedUrl ) {
+	if ( typeof data !== 'undefined' ) {
+		data.feeds.map( function( feed ) {
+			if ( feed.feed_ID ) {
+				feedsByUrl[ feedUrl ] = feed.feed_ID;
+				FeedStore.emit( 'change' );
```

Comment:
Probably only want to emit once, once all the changes have been made, rather than after each alteration. It cuts down on the number of reconciliations react has to perform.

Also having this as a separate store would make the number of entities that are reconciling in reaction to a feed store change smaller.